package.json

```diff
@@ -106,57 +105,7 @@
   "licenses": [
     {
       "type": "MIT",
-      "url": "https://github.com/processhub/processhub-sdk/master/LICENSE"
+      "url": "https://github.com/roXtra/processhub-sdk/master/LICENSE"
```

Comment:
URL does not work, should be https://github.com/roXtra/processhub-sdk/blob/master/LICENSE probably?

Reply:
or https://raw.githubusercontent.com/roXtra/processhub-sdk/master/LICENSE

Reply:
Thanks!